db/dbWriter.js

```diff
@@ -0,0 +1,94 @@
+/* eslint-disable no-unused-vars */
+const fs = require('fs');
+
+const now = Date.now();
+const pool = require('./pool.js');
+
+let fullData = {};
+fs.readFile('./db/sampleData/sampleData.json', 'utf8', async (err, data) => {
+  if (err) throw err;
+  fullData = JSON.parse(data);
+  const { characteristics } = fullData[1];
+  const charKeys = Object.keys(characteristics);
+  for (let j = 0; j < 3; j += 1) {
+    pool.query(`
+        INSERT INTO characteristics (char_name) VALUES ('${charKeys[j]}');`,
+    (error, res) => {
+      if (error) {
+        console.log(error);
+      }
+    });
+  }
+  const primaryInserter = () => {
+    for (let j = 0; j < 10; j += 1) {
+      let insertString = '';
+      for (let i = 1; i <= 100000; i += 1) {
+        if (i === 100000) {
+          insertString += `(
+            ${fullData[i].product_id}, ${fullData[i].rating}, '${fullData[i].summary}', ${fullData[i].recommend}, '${fullData[i].response}', '${fullData[i].body}', '${fullData[i].date}', '${fullData[i].reviewer_name}', ${fullData[i].helpfulness}
+            );`;
```

Comment:
There's probably a cleaner way to make it so you add the ; at the end without having the giant block of values copy pasted, it took me a second to figure out why you had an if else with the same code for both before realizing.